app/Http/Controllers/TaskController.php

```diff
@@ -177,5 +177,41 @@ public function archived(Request $request)
    
     }
 
-   
+   public function getRecurringTask()
+    {           
+        $tasks = $this->taskService->all();
+
+        $getArchive = [];
+        
+        foreach($tasks as $task)
+        {
+            if(isset($task['recurring'])){
+                 if($taskRecurring["recurring"][0] == "True"){
+                    array_push($getArchive, $task);
+                 }
+            }
+        }
+
+        return response()->json([
+            'message' => "Request Success",
+            'data' => $getArchive
+        ],200);
+    }
+    
+    public function createRecurringTask(Request $request, $id)
+    {
+        $taskRecurring = $this->taskService->find($id);
+        $task = [];
+
+        if(isset($taskRecurring["recurring"])){
+            if($taskRecurring["recurring"][0] != "True"){
```

Comment:
I dont really get why you are selecting the first element of the recurring array when you set it to true only